From 78adb18cdc26095dc92b8e034da715698b5d961f Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Thu, 3 Feb 2022 00:52:00 +0800
Subject: [PATCH 31/36] dt-bindings: leds: leds-axp20x: add compatible for
 AXP803

AXP803 is quite similar to AXP813 as of the PMIC part.

Add a compatible string combo with AXP813 as fallback to the charging
LED part.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 Documentation/devicetree/bindings/leds/leds-axp20x.yaml | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/leds/leds-axp20x.yaml b/Documentation/devicetree/bindings/leds/leds-axp20x.yaml
index 79282d55764b..e54fb782cbd6 100644
--- a/Documentation/devicetree/bindings/leds/leds-axp20x.yaml
+++ b/Documentation/devicetree/bindings/leds/leds-axp20x.yaml
@@ -18,7 +18,11 @@ description: |
 
 properties:
   compatible:
-    const: x-powers,axp813-charger-led
+    oneOf:
+      - const: x-powers,axp813-charger-led
+      - items:
+          - const: x-powers,axp803-charger-led
+          - const: x-powers,axp813-charger-led
 
 required:
   - compatible
-- 
2.30.2

